src/components/config_profile/test/smartDeviceLink.ini

```diff
@@ -168,6 +168,10 @@ FrequencyTime = 1000
 ApplicationListUpdateTimeout = 2
 ; Max allowed threads for handling mobile requests. Currently max allowed is 2
 ThreadPoolSize = 1
+; Menu title default text
+MenuTitle = MENU
```

Comment:
@dcherniev Why did you write this line in ApplicationManager section. I think section GLOBAL PROPERTIES is more suitable.